zerver/views/realm.py

```diff
@@ -310,6 +310,10 @@ def update_realm(
             Realm.JITSI_SERVER_SPECIAL_VALUES_MAP,
         )
 
+        if jitsi_server_url is not None and len(jitsi_server_url) > 200:
+            # Django sets max_length of 200 for URLField by default.
+            raise JsonableError(_("Jitsi server URL too long (limit: 200 characters)."))
```

Comment:
Let's avoid adding a translated string here; maybe add a `max_length` parameter to `check_url` and pass that when validating this parameter.

(We might want to make that the default, but it's not reasonable do that this late in the release cycle)

Reply:
Are we fine with such error message for now?

Reply:
That's probably fine, yeah. I don't expect this to come up in practice.

Reply:
Should I just add a `check_capped_url` and use it here (we already have `check_capped_string`), since otherwise we would need to update all other usages of `check_url` as well?

Reply:
Yeah that works for me.